inference-engine/thirdparty/clDNN/src/kernel_selector_helper.cpp

```diff
@@ -764,7 +763,10 @@ void set_params(const program_node& node, kernel_selector::params& params) {
     }
 }
 
-void set_optional_params(const program_impl& program, kernel_selector::optional_params& params) {
+void set_optional_params(const program_node& node, kernel_selector::optional_params& params) {
+    const auto& program = node.get_program();
+
+    params.uniqueID = std::to_string(program.get_id()) + "_"  + node.get_unique_id();
```

Comment:
This uniqueID seems to be required for correct work, so maybe it makes sense to keep it in `params` instead of `optional_params`?

Reply:
@vladimir-paramuzov. thank you for your comment. I just used optional_params because GetEntryPoint has optional_params as parameter. To pass unique_id, we should add more parameter to the GetEntryPoint. If is it okay, I will add the Params to the GetEntryPoint parameter.

Reply:
@vladimir-paramuzov. I will update new version to keep unique_id in params. thanks.

Reply:
@vladimir-paramuzov  I changed codes to keep unique id in Params. could you please review it again?